docs/source/workflomics-benchmarker/benchmarker-usage.rst

```diff
@@ -37,25 +37,25 @@ To run the Workflomics workflows:
 
 
 Command Line Interface (CLI)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+============================
 
 The Workflomics Benchmarker CLI provides a set of commands to benchmark and run workflows generated on the Workflomics platform. This section provides an overview of the available commands and their usage.
 
 Get a list of available CLI commands for Workflomics Benchmarker::
 
-  $ workflomics benchmark --help
-  $ workflomics run --help
+  $ workflomics benchmark--help
```

Comment:
It seems some more find-replace magic is going on here with the missing space; it's in other files also

Reply:
Just fixed it, thanks!